tests/cli/test_cli.py

```diff
@@ -1470,6 +1470,27 @@ def test_run2(ctx):
         [cl, 'run', 'foo:%s' % dir3, 'foo/bar:%s' % dir1, 'cat foo/bar/f1'], expected_exit_code=1
     )
 
+    # Test that cpu_usage and memory_limit are properly populated
+    uuid = _run_command([cl, 'run', "bash -c 'for i in {1..90}; do sleep 1; done'"])
+    state = get_info(uuid, 'state')
+    # Wait until the bundle is running
```

Comment:
Can we just replace this loop with `wait(uuid)`?

Reply:
Oh I thought `wait(uuid)` is waiting for the bundle to finish. If it's waiting for it to start run, yes.

Reply:
Looks like `wait(uuid)` is waiting for the bundle to finish. When a bundle is finished, the `cpu_usage` is 0. So it's probably better to stick with the loop. I can make it as a method like `wait_for_bundle_running` if you like.